From 5a1ab671d7a719b382464f9f678bb13230799488 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:04:27 -0400
Subject: [usb] ohci: no conditional debugging in root hub hadling

Message-id: <1410887463-20674-196-git-send-email-dzickus@redhat.com>
Patchwork-id: 95200
O-Subject: [RHEL7 PATCH 195/591] ohci: no conditional debugging in root hub hadling
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 00b033bb90c1e8838616f342abe6e2f0a56e839f
Author: Oliver Neukum <oneukum@suse.de>
Date:   Mon Nov 18 13:23:00 2013 +0100

    ohci: no conditional debugging in root hub hadling

    With dynamic debugging the selection is done in user space

    Signed-off-by: Oliver Neukum <oneukum@suse.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/host/ohci-hub.c b/drivers/usb/host/ohci-hub.c
index 61705a7..c81c872 100644
--- a/drivers/usb/host/ohci-hub.c
+++ b/drivers/usb/host/ohci-hub.c
@@ -725,10 +725,8 @@ static int ohci_hub_control (
   temp = roothub_portstatus (ohci, wIndex);
   put_unaligned_le32(temp, buf);
 
-#ifndef OHCI_VERBOSE_DEBUG
- if (*(u16*)(buf+2)) /* only if wPortChange is interesting */
-#endif
-  dbg_port (ohci, "GetStatus", wIndex, temp);
+  if (*(u16*)(buf+2)) /* only if wPortChange is interesting */
+   dbg_port(ohci, "GetStatus", wIndex, temp);
   break;
  case SetHubFeature:
   switch (wValue) {
-- 
1.7.1